pkg/cloudprovider/gce/gce.go

```diff
@@ -480,11 +482,14 @@ func (gce *GCECloud) NodeAddresses(_ string) ([]api.NodeAddress, error) {
 	if err != nil {
 		return nil, fmt.Errorf("couldn't get external IP: %v", err)
 	}
+	internalIP, err := gce.metadataAccess(INTERNAL_IP_METADATA_URL)
+	if err != nil {
+		return nil, fmt.Errorf("couldn't get external IP: %v", err)
```

Comment:
And I'd put this above the externalIP code, to be consistent with the ordering everywhere else.